src/NHibernate/MultiTenancy/MultiTenancyStrategy.cs

```diff
@@ -0,0 +1,29 @@
+namespace NHibernate.MultiTenancy
+{
+	/// <summary>
+	/// Strategy for multi-tenancy
+	/// <seealso cref="Cfg.Environment.MultiTenancy"/>
+	/// </summary>
+	public enum MultiTenancyStrategy
+	{
+		/// <summary>
+		/// No multi-tenancy
+		/// </summary>
+		None,
+
+//		/// <summary>
+//		/// Multi-tenancy implemented by use of discriminator columns.
+//		/// </summary>
+//		Discriminator,
+//
+//		/// <summary>
+//		/// Multi-tenancy implemented as separate schemas.
+//		/// </summary>
+//		Schema,
```

Comment:
If you plan to introduce them later, it will shift the enumeration values, which will be a breaking change. So it is better to write down these values explicitly in order to freeze them.

Reply:
I just moved comments to the end.